pkg/volume/git_repo/git_repo_test.go

```diff
@@ -330,11 +330,10 @@ func doTestPlugin(scenario struct {
 			allErrs = append(allErrs,
 				fmt.Errorf("SetUp() failed, volume path not created: %s", path))
 			return allErrs
-		} else {
-			allErrs = append(allErrs,
-				fmt.Errorf("SetUp() failed: %v", err))
-			return allErrs
 		}
+		allErrs = append(allErrs,
```

Comment:
Can you also remove this file from https://github.com/kubernetes/kubernetes/blob/master/hack/.golint_failures so that we don't regress in the future?

Reply:
I just removed it. After that, I found there has some golint issues in this package.

1. don't use an underscore in package name
2. var gitUrl should be gitURL
3. comment on exported function ProbeVolumePlugins should be of the form "ProbeVolumePlugins"

Reply:
@msau42 So should I fix all the golint issues under git_repo packages? Renaming the git_repo package would involve wide range modification.

Reply:
Yes, I think we should try to address all lint errors

Reply:
> I just removed it. After that, I found there has some golint issues in this package.
> 
> 1. don't use an underscore in package name
> 2. var gitUrl should be gitURL
> 3. comment on exported function ProbeVolumePlugins should be of the form "ProbeVolumePlugins"

@msau42 I have already addressed all the left lint error

2, 3 are easy to fix.
But the 1st one is not, remove the underscore in the package name git_repo would involve wide range modification.

Reply:
I think it would be good to try and fix everything. If we keep the pkg in .golint_failures then that can cause future lint failures to be introduced since we're not verifying it.  You can try the changes in a separate commit. If it really gets too complicated, then we can go without it.

Reply:
@msau42  OK, I just fixed all the left issues.